files/common/var/lib/delphix-platform/clear-sharenfs

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/python3
+
+import subprocess
+
+lines = subprocess.check_output(
+    "zfs list -t filesystem -r -H -o name,sharenfs,mounted domain0",
```

Comment:
I'm wondering if it would be faster to just look for any filesystem which has the `sharenfs` property set and the `source` is set to local. For example:

I've set the `sharenfs` property at a higher level and let datasets inherit it below:
```
# zfs list -t filesystem -r -H -o name tank
tank
tank/1
tank/1/2
tank/1/2/3
tank/1/2/3/4
tank/1/2/3/4/5
# zfs get -r -t filesystem -o name -Hs local sharenfs tank
tank/1/2
```

Reply:
Right, that was my feedback as well. I think it's worth doing this given how much boot time this part of the migration workflow is taking up. Anything we can do to squeeze seconds or minutes out of that boot time the better, especially if it's a simple thing like this.